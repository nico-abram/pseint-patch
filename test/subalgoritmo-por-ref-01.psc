SubAlgoritmo x<-foo1(a por referencia)
    si a==nil Entonces
        escribir "num es nil"
    SiNo
        escribir a
    FinSi
    x<-a
FinSubAlgoritmo

Algoritmo Main
    definir n como entero
    n<-1
    definir tmp como entero
  tmp<-foo1(1);escribir tmp
  tmp<-foo1(n-1);escribir tmp
  tmp<-foo1(1-1);escribir tmp
FinAlgoritmo
