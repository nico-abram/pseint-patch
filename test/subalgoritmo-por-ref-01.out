=== Line 14: SynError 268
=== Line 15: SynError 268
=== Line 16: SynError 268
